### PR TITLE
Update introduction.md

### PR DESCRIPTION
**Pull request by TamaraRingas**

### CODE REVIEW

**Comment by TamaraRingas** [comment]
Small grammar fix.

**Comment by pertrai1** [comment]
👍 LGTM

File: exercises/concept/lasagna-master/.docs/introduction.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Introduction
 
-A function allows to group code into a reusable unit.
+A function allows for grouping of code into a reusable unit.
 There are multiple ways to define functions in JavaScript.
 Here we will look at _function declarations_ and _function expressions_..
```
Comment (junedev) [inline exercises/concept/lasagna-master/.docs/introduction.md:5]: This line has an excess period at the end. Maybe you could remove it as well.

File: exercises/concept/lasagna-master/.docs/introduction.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Introduction
 
-A function allows to group code into a reusable unit.
+A function allows for grouping of code into a reusable unit.
```
Comment (junedev) [inline exercises/concept/lasagna-master/.docs/introduction.md:3]: The same text appears in other files that need to same fix. Please follow the first point in the automated comment you got, it explains where to look: https://github.com/exercism/javascript/pull/1537#issuecomment-976494256

File: exercises/concept/lasagna-master/.docs/introduction.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ function divide(a, b) {
 ## Function Expression
 
 A function declaration is a standalone statement.
-But sometimes it is helpful to define a function as part of another expression, e.g., in an assignment, as a function parameter ([callback][concept-callbacks]) or as value in an [object][concept-objects].
+But sometimes it is helpful to define a function as part of another expression, e.g. in an assignment, as a function parameter ([callback][concept-callbacks]) or as value in an [object][concept-objects].
```
Comment (junedev) [inline exercises/concept/lasagna-master/.docs/introduction.md:98]: I am ok with merging this change but please take note of the Exercism guidelines regarding this:
> Many American English style guides state that the abbreviations "i.e." and "e.g." should be followed by a comma [...]. This is permitted, but not required within text on Exercism.

Source: https://exercism.org/docs/building/markdown/style-guide#h-oxford-comma

**Review by junedev — COMMENTED** [review]
Thanks for your contribution. Please make sure to apply the changes consistently.

**Comment by junedev** [comment]
@TamaraRingas Would be great if you could address the comments so this can be merged.

**Comment by TamaraRingas** [comment]
Hi June,

Thank you for the comments, sorry I have been away without internet access.

I addressed the additional extra full stop.

Thanks for letting me know about the e.g., in American english, I did not
know.

I will look into the other places that grammatical mistake happens and fix
them today.

Best regards,
Tamara

**Comment by SleeplessByte** [comment]
@TamaraRingas let us know if / when you would be able to do this, so we can merge these great changes!

File: exercises/concept/lasagna-master/.docs/introduction.md (hunk 1 above)
Comment (junedev) [reply exercises/concept/lasagna-master/.docs/introduction.md:5]: This was fixed in another PR.

**Comment by junedev** [comment]
The issue with the double period has been addressed in another PR. Since the other two changes are "optional" as the current text is not wrong according to our guidelines and it has been another two weeks, I am closing this PR now.

**Pull request closed without merging**